unity-renderer/Assets/DCLPlugins/AvatarModifierAreaFeedbackPlugin/AvatarModifierAreaFeedbackPlugin.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using DCL.AvatarModifierAreaFeedback;
+using UnityEngine;
+
+public class AvatarModifierAreaFeedbackPlugin : IPlugin
+{
+    
+    private AvatarModifierAreaFeedbackController avatarModifierAreaFeedbackController;
+    
+    public AvatarModifierAreaFeedbackPlugin() { avatarModifierAreaFeedbackController = new AvatarModifierAreaFeedbackController(); }
```

Comment:
I would create the view here as well and use dependency injection to pass it to the controller.